<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registro</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../proyectoo/recursos.qrc">
    <normaloff>../imagenes/registro-en-linea.png</normaloff>../imagenes/registro-en-linea.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>541</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>539</width>
       <height>449</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>101</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../proyectoo/recursos.qrc">../imagenes/709011.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>401</width>
        <height>201</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>ID</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Disponibilidad</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Horario de entrada</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Horario de Salida</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 26pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 73, 107);</string>
      </property>
      <property name="text">
       <string>Zona de parking</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>340</y>
        <width>331</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(0, 0, 0);
color: rgb(0, 85, 255);</string>
      </property>
      <property name="digitCount">
       <number>8</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>361</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 43);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>Informacion de usuario</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtnombre">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 85, 127);</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblnombre">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Ebrima&quot;;
color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>Nombre</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtdepartamento">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>180</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 85, 127);</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_departamento">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Ebrima&quot;;
color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>Departamento</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>220</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Ebrima&quot;;
color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>Matricula</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtmatricula">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>220</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 85, 127);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonguardar">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 85, 127);
color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>Guardar registro </string>
     </property>
     <property name="icon">
      <iconset resource="../proyectoo/recursos.qrc">
       <normaloff>../imagenes/salvar.png</normaloff>../imagenes/salvar.png</iconset>
     </property>
     <property name="shortcut">
      <string notr="true"/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonbuscar">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>270</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 85, 127);
color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>Buscar lugares</string>
     </property>
     <property name="icon">
      <iconset resource="../proyectoo/recursos.qrc">
       <normaloff>../imagenes/lupa.png</normaloff>../imagenes/lupa.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>361</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../proyectoo/recursos.qrc">../imagenes/estacionamiento.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources>
  <include location="../proyectoo/recursos.qrc"/>
 </resources>
 <connections/>
</ui>
